sql-plugin/src/main/320until340-non330db/scala/com/nvidia/spark/rapids/shims/ShuffleOriginUtil.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, NVIDIA CORPORATION.
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
Change should be reverted if there are no other changes in this file.

Reply:
File was actually moved into this shim, is the suggestion to keep the old copyright?

Reply:
Yes, if we're not touching the contents of the file then the copyright remains untouched.